indexing/jenkins/index.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -exo pipefail
+
+# Help
+function help(){
+  printf "\n"
+  printf "Usage: export <options> $0\n"
+  printf "\n"
+  printf "Options supported, export them as environment variables:\n"
+  printf "\t ES_SERVER=str,                    str=elasticsearch server url, default: ""\n"
```

Comment:
@chaitanyaenr there're no default defined for several variables

Reply:
Fixed it, thanks.